tests/h/accounts/views_test.py

```diff
@@ -10,6 +10,8 @@
 from h import accounts
 from h.accounts import views
 
+from tests.h import conftest
```

Comment:
You should never need to import the conftest file, and doing so is a surefire recipe for circular imports. Just expose the things you want to use in tests as fixtures.